core/server/worker/src/test/java/alluxio/worker/block/SpaceReserverTest.java

```diff
@@ -66,7 +65,6 @@ public void after() {
     mExecutorService.shutdownNow();
   }
 
-  @Ignore("https://alluxio.atlassian.net/browse/ALLUXIO-2908")
   @Test
   public void reserveCorrectAmountsOfSpace() throws Exception {
     BlockWorker blockWorker = PowerMockito.mock(BlockWorker.class);
```

Comment:
Please replace all `PowerMockito.mock` to `Mockito.mock` to avoid using `PowerMockito`. 
In general in this codebase, we are trying to use `Mockito` library for mocking, rather than `PowerMockito`